services/github/auth/acceptor.spec.js

```diff
@@ -43,7 +43,10 @@ describe('Github token acceptor', function() {
   })
```

Comment:
I couldn't add this above, but refers to top level describe block on line 14:

Can't we use sinon here via:
```js
before(function() {
    // make sure properties exist
    serverSecrets['gh_client_id'] = undefined
    serverSecrets['shields_ips'] = undefined
    sinon.stub(serverSecrets, 'gh_client_id').value(fakeClientId)
    ...
  })
```

I'm not sure if that buys us anything, but we've done the same elsewhere in other services (for example https://github.com/badges/shields/blob/master/services/jira/jira-test-helpers.js#L10)